mycroft/audio/main.py

```diff
@@ -449,6 +449,28 @@ def setup_pulseaudio_handlers(self, pulse_choice=None):
                 self.pulse_quiet = self.pulse_lower_volume
                 self.pulse_restore = self.pulse_restore_volume
 
+    def shutdown(self):
+        for s in self.service:
+            LOG.info('shutting down ' + s.name)
+            s.shutdown()
```

Comment:
Add a try-except block here so a failure in one backend won't cause others to not shutdown.